app/models/lockstep/user.rb

```diff
@@ -7,4 +7,21 @@ class Lockstep::User < Lockstep::ApiRecord
   belongs_to :account, class_name: "Lockstep::Account", loader: ->(user) { Lockstep::Account.first }
 
   alias_attribute :last_logged_in, :b2_c_last_logged_in
+
+  def self.invite(email_addresses)
+    emails = email_addresses.map { |email| { email: email } }
+    resp = resource.post('/invite', body: emails, params: {})
+    parsed_response = JSON.parse(resp.body)
+    if resp.code != '200'
+      if resp.code == '401'
```

Comment:
Bro do use case statements here if possible. 

Suggestion: One better way what I could think of is making this as a map.
{ 400 => Lockstep::Exceptions::BadRequestError, ... } so that in future anybody can reference this map using the status code and raise the error in one single line rather than having a case statement.
@vivekagb if the second option seems viable, where can we store this map, so that it can be accessible on all models?

Reply:
We don't need 2 if conditions like 
if not of 200
  if == 400

Lets just make this into a single case statement

Reply:
Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/222d8dc8e1c323a079d26b7c283b3f67aed9af14#diff-05cb5ee86320f350c673545a725c3678dde1f823bfb52d0ed7e400a77447d250)